site/content/contribute/devtalks/_index.md

```diff
@@ -1,12 +1,12 @@
-+++
+---
 title = "Mattermost Dev Talks"
```

Comment:
I think you also need to change the `=`s to `:`s, so that's it YAML formatted

Reply:
Your right. I will fix this.